### ISSUE TITLE
Remote Explorer not showing SSH targets

### ISSUE THREAD

**Issue by plwalsh** [microsoft/vscode-remote-release#7644]
```
Version: 1.74.0-insider (user setup)
Commit: 5235c6bb189b60b01b1f49062f4ffa42384f8c91
Date: 2022-12-05T10:17:57.998Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: Yes
```

After a recent update, I noticed the *Remote Explorer* viewlet no longer shows an option for viewing "SSH Targets". The dropdown at the top only displays options for "Containers" and "WSL Targets". I use this panel constantly for switching between various SSH workspaces, so its disappearance was immediately apparent.

I believe the issue appeared sometime last week, which would align with with the extension's "Last released" date of 11/28/2022.

Steps to Reproduce:

1. Open *Remote Explorer* viewlet.
2. Click dropdown at top.
3. Observe lack of "SSH Targets" option.

**Comment by plwalsh** [comment]
Hi @eleanorjboyd! The version of remote-ssh I have is `v0.92.0`. I do not have a separate "remote-explorer" extension installed. Is that new? I did not need that previously.

**Comment by eleanorjboyd** [comment]
Yes there is actually a remote explorer extension (you can find it in the marketplace under that name). I should be installed along with your remote-ssh extension. Can you confirm it says installed for that extension and the version? Thanks!

**Comment by plwalsh** [comment]
I've had the remote-ssh extension installed for a long time now&mdash;well before the remote-explorer extension ever existed. Should the remote-explorer extension have gotten installed automatically when it was first published? It does not appear like it did:

![image](https://user-images.githubusercontent.com/43576448/206028818-ab251265-010e-4ae7-bc95-b4662bfe5473.png)

I was able to install it manually (`v0.0.2`), and that brought the "SSH Targets" back into the *Remote Explorer* viewlet. It seems odd, though, that the functionality was moved into a "child" extension, but that child extension wasn't installed automatically, meaning the functionality was just removed unknowingly?
